### PR TITLE
[LL-7343] added stake button to eth account

### PR DESCRIPTION
**Pull request by ghost**
## 🦒 Context (issues, jira)
added stake button to eth account

https://ledgerhq.atlassian.net/browse/LL-7343

Based on https://github.com/LedgerHQ/ledger-live-desktop/pull/4240

~Blocked by https://github.com/LedgerHQ/ledger-live-desktop/pull/4496~

## 💻  Description / Demo (image or video)

<details>
<summary> 🎥  </summary>

https://user-images.githubusercontent.com/87011321/147817580-0dc0936f-cd67-47ae-ab80-41fcae71ff34.mov

</details>

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [x] a specific test planned is defined on Jira
  - [ ] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

### CODE REVIEW

**Review by LFBarreto — DISMISSED** [review]

**Review by OlivierFreyssinet — DISMISSED** [review]

**Comment by ofreyssinet-ledger** [comment]
LGTM 👌
One thing that I'm curious about is whether there is a particular reason why we're doing Ethereum specific stuff in this generic `AccountHeaderAction` instead of having a family specific `ethereum/AccountHeaderManageActions` like for other coin families such as Algorand or Polkadot?

**Comment by achabrolin-ledger** [comment]
> LGTM 👌 One thing that I'm curious about is whether there is a particular reason why we're doing Ethereum specific stuff in this generic `AccountHeaderAction` instead of having a family specific `ethereum/AccountHeaderManageActions` like for other coin families such as Algorand or Polkadot?

Good question indeed, the answer is that, as it is the case for WalletConnect, this feature is only available for the Ethereum blockchain and not other Ethereum family blockchains (like Binance Smart Chain for example)

**Comment by ofreyssinet-ledger** [comment]
> > LGTM 👌 One thing that I'm curious about is whether there is a particular reason why we're doing Ethereum specific stuff in this generic `AccountHeaderAction` instead of having a family specific `ethereum/AccountHeaderManageActions` like for other coin families such as Algorand or Polkadot?
> 
> Good question indeed, the answer is that, as it is the case for WalletConnect, this feature is only available for the Ethereum blockchain and not other Ethereum family blockchains (like Binance Smart Chain for example)

Ah ok this makes sense, thanks for the explanation. So as I understand we currently have a logic for family specific features but not blockchain specific features?

File: src/renderer/components/Button.js [hunk 1]
```diff
@@ -280,6 +280,7 @@ export const Base: ThemedComponent<*> = styled.button.attrs(p => ({
   cursor: ${p => (p.disabled ? "not-allowed" : "pointer")};
   height: ${p => (p.small ? 34 : 40)}px;
   pointer-events: ${p => (p.disabled ? "none" : "")};
+  width: ${p => (p.fullWidth ? "100%" : "auto")};
```
Comment (ggilchrist-ledger) [inline src/renderer/components/Button.js:283]: Out of interest what is this doing? @achabrolin-ledger

Comment (achabrolin-ledger) [reply src/renderer/components/Button.js:283]: Basically it allow (via the `fullWidth` prop) to specify if the element should take the full with of his parent element (`100%`) or not. In the later case, the width would be calculated automatically by the browser due to the `auto` value

cf. https://developer.mozilla.org/en-US/docs/Web/CSS/width#values

**Pull request merged**